spec/system/visit_user_journey_view_certificate_page_spec.rb

```diff
@@ -43,13 +44,44 @@
       click_link 'Replace certificate'
       expect(current_path).to eql before_you_start_path(sp_component.component_type, sp_component.id, sp_component.encryption_certificate_id)
     end
+
     it 'signing certificate information and navigates to next page' do
-      certificate = create(:sp_signing_certificate)
-      sp_component = certificate.component
+      sp_signing_certificate = create(:sp_signing_certificate)
+      sp_component = sp_signing_certificate.component
       visit view_certificate_path(sp_component.component_type, sp_component.id, sp_component.signing_certificates[0])
       expect(page).to have_content 'Verify Service Provider: signing certificate'
       click_link 'Add new certificate'
       expect(current_path).to eql before_you_start_path(sp_component.component_type, sp_component.id, sp_component.signing_certificates[0])
     end
   end
+
+  context 'show signing' do
+    it 'specific information when certificate is primary' do
+      first_certificate = create(:msa_signing_certificate, component: create(:msa_component, team_id: @user.team))
+      defaults = {
+        usage: CERTIFICATE_USAGE::SIGNING,
+        value: second_certificate,
+        component: first_certificate.component
+      }
+      second_cert = Certificate.create(defaults)
+      visit root_path
+      click_link 'Signing certificate (primary)'
+      expect(page).to have_content 'GOV.UK Verify is adding your certificate to its configuration'
+      expect(page).to_not have_button('Add new certificate')
+    end
+
+    it 'specific information when certificate is secondary' do
+      first_certificate = create(:msa_signing_certificate, component: create(:msa_component, team_id: @user.team))
+      defaults = {
+        usage: CERTIFICATE_USAGE::SIGNING,
+        value: second_certificate,
+        component: first_certificate.component
+      }
+      second_cert = Certificate.create(defaults)
+      visit root_path
+      click_link 'Signing certificate (secondary)'
+      expect(page).to have_content 'Wait for an email from GOV.UK Verify confirming your new signing certificate is in use'
+      expect(page).to_not have_button('Add new certificate')
+    end
```

Comment:
I think maybe the first it could be with just one certificate and check that the Add button is there. Then the second it could be as it is

Reply:
I thought that but the other tests above these new ones already facilitate that functionality. Otherwise it would be duplicated expectations and the tests would be identical. The tests above already tests that you can click a button once a certificate has been added and it takes you to the next page